modules/fundamental/src/advisory/endpoints/test.rs

```diff
@@ -49,33 +44,19 @@ where
     actix_web::test::call_and_read_body_json(app, req).await
 }
 
-async fn ingest(service: &IngestorService, data: &[u8]) -> IngestResult {
-    use trustify_module_ingestor::service::Format;
-    service
-        .ingest(
-            ("source", "unit-test"),
-            Some("Capt Pickles Industrial Conglomerate".to_string()),
-            Format::from_bytes(data).unwrap(),
-            ReaderStream::new(data),
-        )
```

Comment:
If you change the parameter of `ingest_documents` to a collection of &[u8]'s? You can use this logic to detect the `Format` and create a `Stream`...